advanced/Scripts/webpage.sh

```diff
@@ -176,6 +176,10 @@ trust-anchor=.,19036,8,2,49AAC11D7B6F6446702E54A1607371607A1A41855200FD2CE1CDDE3
 		# Listen only on one interface
 		add_dnsmasq_setting "interface" "${PIHOLE_INTERFACE}"
 	fi
+	if [[ "${CONDITIONAL_FORWARDING}" == true ]]; then
+		add_dnsmasq_setting "server=/${args[8]}/${args[7]}"
+		add_dnsmasq_setting "server=/${args[9]}/${args[7]}"
```

Comment:
These should use variables from setupVars instead of arguments.

Reply:
I'm sorry, I dont understand.  Isn't this how the setupVars file gets updated?  savesettings.php calls the pihole script which passes these arguments to webpage.sh which will write to setupVars.conf.

Reply:
`ProcessDNSSettings` is called after the `CONDITIONAL_FORWARDING` settings are changed, so use the `CONDITIONAL_FORWARDING_IP` and `CONDITIONAL_FORWARDING_DOMAIN` variables instead of the `args` variables. With the `args` variables, conditional forwarded might be enabled but incorrectly set because `ProcessDNSSettings` was called without those `args` set.